native.toml

```diff
@@ -0,0 +1,121 @@
+# Usage:
+# databend-query -c databend_query_config_spec.toml
```

Comment:
I think this file should not be placed in the root directory.

Reply:
Oh, i forgot to remove this file.